README.adoc

```diff
@@ -165,19 +165,18 @@ include::complete/src/main/java/com/example/securingweb/WebSecurityConfig.java[]
 ====
 
 The `WebSecurityConfig` class is annotated with `@EnableWebSecurity` to enable Spring
-Security's web security support and provide the Spring MVC integration. It also extends
-`WebSecurityConfigurerAdapter` and overrides a couple of its methods to set some specifics
-of the web security configuration.
+Security's web security support and provide the Spring MVC integration. It also exposes
+two beans to set some specifics of the web security configuration.
```

Comment:
Can you be more detailed about what specifics are exposed?

Also, "of the web security configuration" should be "for the web security configuration".

Thanks.

Reply:
> Can you be more detailed about what specifics are exposed?
It is explained in the two following paragraphs, I don't think we need to repeat it here.